### PR TITLE
1062 add placeholder text for input text fields

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: templates/events/add-new-event.html
Comment (sandeep-ps) [inline templates/events/add-new-event.html:1]: @bingzhang, Could you please check this file? There are some repeated entries when there are changes, changes to button CSS classes (not sure if that was intentional). Also, it looks like not all changes are in place.

File: templates/events/event-edit.html [hunk 1]
```diff
@@ -977,7 +977,7 @@ <h5>Display Only with Super Event</h5>
                             <div class = "form-group row" >
                                   <div class="form-group col-sm-3 offset-sm-2">
                                         {% if sub.name %}
-                                            <input class="form-control search-id" id="sub-id" name="name" type="text" placeholder="Enter a keyword of subevent title, and select the subevent from the dropdown list" value="{{sub.name}}" />
+                                            <input class="form-control search-id" id="sub-id" name="name" type="text" placeholder="Search sub-events by words in the title." value="{{sub.name}}" />
```
Comment (sandeep-ps) [inline templates/events/event-edit.html:980]: The sub-events placeholder text is in multiple places. The value has to be changed to "Search sub-events by title."

File: templates/events/add-new-event.html
Comment (bingzhang) [reply templates/events/add-new-event.html:1]: css change is necessary.

File: templates/events/event-edit.html (hunk 1 above)
Comment (bingzhang) [reply templates/events/event-edit.html:980]: I checked again. I did not find other places. Can you take another look?

File: templates/events/add-new-event.html [hunk 2]
```diff
@@ -120,7 +120,7 @@ <h5>Category<span style="color: red">*</span></h5>
             </div>
             <div class="col-sm-9">
               <div class="subcategory-menu eventsmanager-dropdown-menu">
-                <input type="text" id="dropdownmenu-subcategory" name="subcategory" class="form-control dropdown-menu-inputfield" placeholder="Choose an event subcategory (disabled) ..." readonly>
+                <input type="text" id="dropdownmenu-subcategory" name="subcategory" class="form-control dropdown-menu-inputfield" placeholder="Choose a subcategory (disabled) ..." readonly>
```
Comment (sandeep-ps) [inline templates/events/add-new-event.html:123]:
```suggestion
                <input type="text" id="dropdownmenu-subcategory" name="subcategory" class="form-control dropdown-menu-inputfield" placeholder="Choose a subcategory (disabled)." readonly>
```

File: templates/events/event-edit.html (hunk 1 above)
Comment (sandeep-ps) [inline templates/events/event-edit.html:980]:
```suggestion
                                            <input class="form-control search-id" id="sub-id" name="name" type="text" placeholder="Search sub-events by title." value="{{sub.name}}" />
```

Comment (sandeep-ps) [reply templates/events/event-edit.html:980]: Provided additional in-line suggestions. Resolving this comment.

**Review by sandeep-ps — APPROVED** [review]
The changes look good. Approving. Thanks.

**Pull request merged**